.github/workflows/test.yml

```diff
@@ -15,10 +15,8 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  #### TEST STAGE ####
+  #### TEST JOB ####
   test:
-    runs-on: ubuntu-latest
-
```

Comment:
OS version is strategy/matrix dependent: goes _after_ the matrix.

Reply:
Except it isn't (in the matrix), so it doesn't matter. And even if it was, the way the scripts are read by GH, you can still refer to the matrix even when the directive is _above_ the matrix definition, so this is an unnecessary change.

Reply:
I know, I know... when you think about code (e.g. YAML code) it is much easier to follow line in chronological order.

Reply:
Agreed, but that would only come into play _if_ at some point the OS would be added to the matrix and this change would become relevant in that case. But not now.